views/home.hbs

```diff
@@ -1,8 +1,4 @@
-<header class="vh-100 w-100 pv6 ">
-    <a class="" href='/businessType'>
-      <img class="" src="./assets/logos/easy_naz_logo_splash.svg" alt="Picture of easyNAZ Logo">
-    </a>
-    <div class="fixed bottom-0 w-100 tc">
-      <img class="w3 pv4" src="./assets/logos/trans-al-logo.png" alt="Picture of almanarah Logo">
-    </div>
-</header>
+<body>
```

Comment:
the {{body}} call from the layout/main.hbs is already within ```<body>``` tags so I don't think that these two are necessary